### PR TITLE
display list subscription

### PR DESCRIPTION
**Pull request by faultables**
in this updates I add functionality to list subscriptions

### CODE REVIEW

**Comment by faultables** [comment]
@anakmagang mengapa PR si @ri7nz dapet deployment URL tapi gue kagak?

File: app/src/fragments/Shell.js [hunk 1]
```diff
@@ -1,38 +1,114 @@
 import React, { Component } from 'react'
 import Fab from '@material-ui/core/Fab'
 import AddIcon from '@material-ui/icons/Add'
+import Card from '@material-ui/core/Card'
+import CardActionArea from '@material-ui/core/CardActionArea'
+import CardHeader from '@material-ui/core/CardHeader'
+import Avatar from '@material-ui/core/Avatar'
+import SimpleIcons from 'simple-icons-react-component'
+import { navigate } from '@reach/router'
```
Comment (r17x) [inline app/src/fragments/Shell.js]: Check your linter for warning import and never use

File: app/src/fragments/Shell.js [hunk 2]
```diff
@@ -1,38 +1,114 @@
...
+
+import { checkLogin } from '../services/user'
```
Comment (r17x) [inline app/src/fragments/Shell.js]: ⚠️ import & never use.

**Review by r17x — CHANGES_REQUESTED** [review]

File: app/src/fragments/Shell.js (hunk 1 above)
Comment (faultables) [reply app/src/fragments/Shell.js]: I'm sorry I believe this happen while I merge with previous commit, let me fix it

File: app/src/fragments/Shell.js (hunk 2 above)
Comment (faultables) [reply app/src/fragments/Shell.js]: Screen Shot 2019-06-26 at 20 57 20

**Review by r17x — APPROVED** [review]

**Comment by faultables** [comment]
Thank you!

**Pull request merged**